lib/oxigraph/Cargo.toml

```diff
@@ -31,6 +31,7 @@ digest.workspace = true
 hex.workspace = true
 json-event-parser.workspace = true
 md-5.workspace = true
+openssl-sys = "0.9.102"
```

Comment:
This will try compile openssl-sys for all plateforms, includings ones where we use the host native SSL implementation if the `http-client-native-tls` feature is enabled (macOS, Windows) and WASM where openssl can't be compiled. I would tend to think it would be better to dynamically link to libressl on OpenBSD, it [seems to work](https://github.com/sfackler/rust-openssl/issues/2050) or, if not, only add this dependency if the target is openbsd and the `http-client-native-tls` feature is enabled.

Reply:
That makes perfect sense. I will investigate the dynamic linking approach on OpenBSD and try that instead.

I have to research this a bit further, but it seemed that an older version of openssl-sys was being used in the build. I have to track down how that happened without it being listed in Cargo.toml or Cargo.lock files. I could have misinterpreted the build errors at the time.

Reply:
Thank you! This is weird, I see `openssl-sys` in [the lock file](https://github.com/oxigraph/oxigraph/blob/196526f01e3aa12999a9160255b94200cdae22b0/Cargo.lock#L1010). It is a dependency of `native-tls` that is itself a dependency of `oxhttp` if the `native-tls` feature is enabled (this feature is enabled by the `http-client-native-tls` feature of `oxigraph`)